packages-exp/auth-exp/src/core/strategies/email_and_password.test.ts

```diff
@@ -391,16 +392,20 @@ describe('core/strategies/email_and_password/createUserWithEmailAndPassword', ()
 
   it('should sign in the user', async () => {
     const {
-      credential,
+      _tokenResponse,
       user,
       operationType
-    } = await createUserWithEmailAndPassword(
+    } = (await createUserWithEmailAndPassword(
```

Comment:
nit: why the extra paren?

Reply:
The formatter put it in. I think because it gets cast now